gitpod-io/gitpod
# Remove CDE Universe footnote

## Description

This will remove the [CDE Universe](https://cdeuniverse.com/) footnote added in https://github.com/gitpod-io/gitpod/pull/17501. 🪐 

The conference will be be over by the time we merge and deploy this change.

## How to test
Create a new workspace and notice there's no footnote on the workspace start page.

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - gt-remote-6eeab5ed4c</li>
	<li><b>🔗 URL</b> - <a href="https://gt-remote-6eeab5ed4c.preview.gitpod-dev.com/workspaces" target="_blank">gt-remote-6eeab5ed4c.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>
 
/hold
